doc/src/properties/frame.toml

```diff
@@ -36,9 +36,15 @@ CML = "The text of the ``title`` attribute in a molecule node. Only set if the a
 [time]
 type = "number"
 
+LAMMPS = "The time of the frame in time units."
 TRR = "The time of the frame in pico seconds."
 XTC = "The time of the frame in pico seconds."
 
+[units]
```

Comment:
Since this is specific to LAMMPS, we may want to call this property `lammp_units`. This way we don't end up using any `units` property defined by another format when writing a LAMMPS file.

Reply:
Changed to `lammps_units`.